src/main/java/stringcalculator/StringCalculater.java

```diff
@@ -0,0 +1,62 @@
+package stringcalculator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class StringCalculater {
+
+    private final List<Double> numbers = new ArrayList<>();
+    private final List<String> operatorList = new ArrayList<>();
+    private double result;
+
+    public StringCalculater(String input) {
+        validateInput(input);
+        String[] strings = splitInput(input);
+        result = Double.parseDouble(strings[0]);
+        for (int i = 0; i < strings.length; i++) {
+            separateInput(strings, i);
+        }
+    }
+
+    private void separateInput(String[] strings, int i) {
+        if(i%2==1){
```

Comment:
```suggestion
        if (i % 2 == 1) {
```
사소하지만 컨벤션을 지키는 것이 좋습니다.